src/views/Layout.jsx

```diff
@@ -21,15 +21,24 @@ export function Layout() {
 					<Outlet />
 				</main>
 				<nav className="Nav">
-					<a href="#" className="Nav-link">
+					{/* <a href="#" className="Nav-link">
 						Home
 					</a>
 					<a href="#" className="Nav-link">
 						List
 					</a>
 					<a href="#" className="Nav-link">
 						Add Item
-					</a>
+					</a> */}
+					<NavLink to="/" className="Nav-link">
+						Home
+					</NavLink>
+					<NavLink to="/list" className="Nav-link">
+						List
+					</NavLink>
+					<NavLink to="/add-item" className="Nav-link">
+						Add Item
+					</NavLink>
```

Comment:
Great job here, but let's make sure we're not doing extra work outside of the acceptance criteria listed on the ticket - this is something that Kaitlin and Patrick worked on this week! Please undo these changes to make sure we're not overwriting their work 🙏 

- [ ] revert changes related to navlinks

Reply:
done